Generate docs (#2356)
kyma-project/cli: docs/user/gen-docs/01-140-kyma_alpha_module_unmanage.md

# kyma alpha module unmanage

> Unmanage module

Reviewer: ```suggestion
Sets a module to the unmanaged state.
```

## Synopsis

> Use this command to unmanage an existing module

Reviewer: ```suggestion
Use this command to set an existing module to the unmanaged state.
```

```bash
kyma alpha module unmanage <module> [flags]
```

## Flags inherited from parent commands

```bash
  -h, --help                    Help for the command
      --kubeconfig string       Path to the Kyma kubeconfig file
      --show-extensions-error   Print possible error when fetching extensions failed
```

## See also

> * [kyma alpha module](kyma_alpha_module.md) - Manage kyma modules

Reviewer: ```suggestion
* [kyma alpha module](kyma_alpha_module.md) - Manages kyma modules.
```
